pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -165,14 +192,14 @@ buildStdenv.mkDerivation ({
 
   NIX_CFLAGS_COMPILE = toString [
     "-I${glib.dev}/include/gio-unix-2.0"
-    "-I${nss.dev}/include/nss"
+    "-I${nss_pkg.dev}/include/nss"
   ];
 
   MACH_USE_SYSTEM_PYTHON = "1";
 
   postPatch = ''
     rm -rf obj-x86_64-pc-linux-gnu
-  '' + lib.optionalString pipewireSupport ''
+  '' + lib.optionalString (pipewireSupport && lib.versionOlder ffversion "83") ''
```

Comment:
@worldofpeace & @mweinelt can you also test pipewire on the esr build? If it didn't and does not work we can remove this section (and the entire old patch).

Reply:
Building everything as we speak.

Reply:
Pipewire works on 78.5.0esr, had linker problems on the firefox 83.0 build, retrying that right now.

Reply:
Make sure your /tmp has enough inodes as otherwise the build fails frequently.

Reply:
Doesn't work with Firefox 83.0 unfortunately. When trying to share my screen I only see "Use operating system settings". I see no reaction in the pipewire unit.

Reply:
*shrug* shall we disable it then? I have zero clues how to debug this.

Reply:
Go ahead.

Reply:
I am tempted to keep them enabled as they are now.. Things will not work entirely but maybe it is also an environmental thing and works for someone else.